kotlin/internal/jvm/compile.bzl

```diff
@@ -694,15 +694,19 @@ def _run_kt_java_builder_actions(
         if java_info.outputs.jdeps:
             jdeps.append(java_info.outputs.jdeps)
 
-    _run_merge_jdeps_action(
-        ctx = ctx,
-        toolchains = toolchains,
-        jdeps = jdeps,
-        deps = compile_deps.deps,
-        outputs = {
-            "output": output_jdeps,
-        },
-    )
+    if jdeps:
```

Comment:
We could alternatively provide empty jdeps and run the merge action anyways, which would help clean up the conditional-nesting here.

I personally prefer this approach though, as it skips spinning up the merge action entirely.